.travis.yml

```diff
@@ -68,18 +64,12 @@ jobs:
       distributions: sdist bdist_wheel
 
   - stage: test
-    python: 2.7
+    python: 3.9
     env: TOXENV=update-pycodestyle
-    arch: ppc64le
+    arch: ppc64le    
```

Comment:
looks there are extra spaces at the end, please remove them

Reply:
removed the spaces. Thanks!

Reply:
Also, re-iterating over questions asked in this comment https://github.com/kubernetes-client/python/pull/1468#issuecomment-841440671 :

Q 2: Do I require to add the `update-pycodestyle.sh` check for all the current set of python3 versions ( py3{5,6,7,8,9} )?

Q 3: What's the difference between py3{5,6,7,8,9} & py3{5,6,7,8,9}-functional?